packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -76,20 +78,19 @@ abstract class CameraPlatform extends PlatformInterface {
     throw UnimplementedError('prepareForVideoRecording() is not implemented.');
   }
 
-  /// Start a video recording and save the file to [path].
+  /// Starts a video recording
   ///
-  /// A path can for example be obtained using
-  /// [path_provider](https://pub.dartlang.org/packages/path_provider).
+  /// If no [file] parameter is provided, the recording will be saved to a new file in a temporary location.
   ///
   /// The file is written on the flight as the video is being recorded.
-  /// If a file already exists at the provided path an error will be thrown.
-  /// The file can be read as soon as [stopVideoRecording] returns.
-  Future<void> startVideoRecording(int cameraId, String path) {
+  /// If a file already exists at the path for the provided file instance, an error will be thrown.
+  /// The file can be read as soon as [stopVideoRecording] returns it.
```

Comment:
nit: Wrap the documentation at 80 characters:

```suggestion
  /// If a file already exists at the path for the provided file instance, an 
  /// error will be thrown.
  /// The file can be read as soon as [stopVideoRecording] returns it.
```